Public/Src/FrontEnd/Core/FrontEndHostController.cs

```diff
@@ -1441,6 +1474,20 @@ private static string ConstructProgressRemainingMessage(TimeSpan elapsed, IReadO
                 : "0";
         }
 
+        private static string ConstructProgressRemainingMessage(TimeSpan elapsed, IReadOnlyCollection<(PipGraphFragmentSerializer, Task<bool>)> remainingItems)

```

Comment:
I don't think this method is necessary, because it looks pretty much the same as the one above it.

What I would do instead is:
  1. change the type of the `remainingItems` arg in the `ConstructProgressRemainingMessage` method above from `IReadOnlyCollection<ModuleEvaluationProgress>` to `IEnumerable<string>`
  2. call this new method from both `LogFragmentEvaluationProgress` and `LogModuleEvaluationProgress`